src/components/molecules/AssetStatus.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { ReactElement } from 'react'
+import Status from '../atoms/Status'
+import Tooltip from '../atoms/Tooltip'
+import styles from './AssetStatus.module.css'
+
+export default function AssetStatus({
+  isOrderDisabled
+}: {
+  isOrderDisabled: boolean
+}): ReactElement {
+  return isOrderDisabled === true ? (
+    <div className={styles.wrapper}>
+      <Status className={styles.status} state="error" />
+      <span className={styles.text}>Disabled</span>
```

Comment:
this still does not account for changes in 2 other PRs which uses the same component. Just make it accept a `message: string` & `status: 'Disabled' | 'Unavailable'` prop and be done with it, make all changes in this component then based on those props if needed. The rest of the logic should happen outside in the parent component where component is used.

Reply:
yes thats what I am having in my mind too.
But ocean.js returned `status` is `number` here, do we modify ocean.js or handle that in market with a `if else`?
https://github.com/oceanprotocol/ocean.js/blob/02693021b1d5195ad4c5e5b0d64210c96fd0be72/src/ocean/Assets.ts#L756-L774

Reply:
sorry, what do you mean with ocean.js? I mean to pass a `message` text which then is put as the tooltip. And the `status` is what's being output on screen by default, instead of the hardcoded copy in the line I added this comment to. Nothing involved with ocean.js

Reply:
Take a look at this too https://github.com/oceanprotocol/market/issues/685